cupy/polynomial/polyutils.py

```diff
@@ -75,3 +75,24 @@ def as_series(alist, trim=True):
     dtype = cupy.common_type(*arrays)
     ret = [a.astype(dtype, copy=False) for a in arrays]
     return ret
+
+
+def _add(c1, c2):
```

Comment:
Why is this function placed in this file? This function is used only from `chebyshev.py`?

Reply:
Anyway, this implementation looks duplicated with polyadd.

Reply:
> Why is this function placed in this file? This function is used only from `chebyshev.py`?

`Hermite`, `HermiteE`, `Laguerre`, `Legendre` and `Power` series uses the same `_add` and `_sub` 

> this implementation looks duplicated with polyadd.

Yes but with the reverse order of the Poly1d convention, instead of reversing twice using `polyadd/sub`

Reply:
How about reversing the inputs before the function call?